WordPress/Classes/ViewRelated/NUX/Login2FAViewController.swift

```diff
@@ -1,40 +1,279 @@
 import UIKit
+import SVProgressHUD
+import WordPressComAnalytics
+import WordPressShared
+
+class Login2FAViewController: NUXAbstractViewController, SigninWPComSyncHandler, SigninKeyboardResponder, LoginViewController {
+    @IBOutlet weak var instructionLabel: UILabel!
+    @IBOutlet weak var verificationCodeField: LoginTextField!
+    @IBOutlet weak var sendCodeButton: UIButton!
+    @IBOutlet weak var submitButton: NUXSubmitButton!
+    @IBOutlet weak var statusLabel: UILabel!
+    @IBOutlet var bottomContentConstraint: NSLayoutConstraint?
+    @IBOutlet var verticalCenterConstraint: NSLayoutConstraint?
+    var pasteboardBeforeBackground: String? = nil
+
+    lazy fileprivate var loginFacade: LoginFacade = {
+        let facade = LoginFacade()
+        facade.delegate = self
+        return facade
+    }()
+
+    override var sourceTag: SupportSourceTag {
+        get {
+            return .wpComLogin
+        }
+    }
 
-class Login2FAViewController: Signin2FAViewController, LoginViewController {
-    // let the storyboard's style stay
-    override func setupStyles() {}
 
-    override func dismiss() {
-        self.performSegue(withIdentifier: .showEpilogue, sender: self)
+    class func controller(_ loginFields: LoginFields) -> Login2FAViewController {
```

Comment:
Oopsie... I had added that while testing the UI (I wired a short cut from the prologue) but forgot to remove it.  I'll give it the axe.